metric/manager.go

```diff
@@ -0,0 +1,61 @@
+package metric
+
+import (
+	"log"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	namespace = ""
+	subsystem = "wit"
+)
+
+var (
+	reqCnt = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Namespace: namespace,
+		Subsystem: subsystem,
+		Name:      "requests_total",
+		Help:      "Counter of requests received into the system.",
+	}, []string{"method", "entity"})
+
+	reqSuccessDuration = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Namespace: namespace,
+		Subsystem: subsystem,
+		Name:      "request_successful_duration_seconds",
+		Help:      "Bucketed histogram of processing time (s) of successfully completed requests, by method (GET/PUT etc.).",
+		Buckets:   prometheus.ExponentialBuckets(0.1, 2, 7),
+	}, []string{"method"})
+)
+
+func init() {
+	reqCnt = register(reqCnt, "requests_total").(*prometheus.CounterVec)
+	reqSuccessDuration = register(reqSuccessDuration, "request_successful_duration_seconds").(*prometheus.HistogramVec)
```

Comment:
Think this should be "request_duration_seonds with a "Status Code' label

Reply:
@aslakknutsen As decided in sync, for both metrics will have same labels.
method = hold http method
entity = controller name
code = http status code (in form of 2xx, 3xx etc.)